job-server/src/main/resources/application.conf

```diff
@@ -124,6 +129,26 @@ spark {
 akka {
   # Use SLF4J/logback for deployed environment logging
   loggers = ["akka.event.slf4j.Slf4jLogger"]
+
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
+  }
+
+  remote {
+    log-remote-lifecycle-events = off
+    netty.tcp {
+      hostname = "127.0.0.1"
+      port = 0
+      send-buffer-size = 512000b
+      receive-buffer-size = 512000b
+      maximum-frame-size = 256000b
+    }
+  }
+
+  cluster {
+    auto-down-unreachable-after = 20s
+    metrics.enabled = off
```

Comment:
:( no adaptive load balancing router? To chatty for this I'm sure and a user can override.